hardhat.config.ts

```diff
@@ -38,13 +44,15 @@ task(
   // Accountant default config
   await hre.run("config:accountant", {
     contract: accountant,
```

Comment:
I believe in the non-zksync deployment we can leave out `contract` and only include `contractAddress`